report_templates/registered_hosts.erb

```diff
@@ -0,0 +1,28 @@
+<%#
+name: Registered hosts
+snippet: false
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    for report on all available hosts.
+  advanced: false
+model: ReportTemplate
+require:
+- plugin: katello
+￼ version: 3.9.0
+-%>
+<%- load_hosts(search: input('Hosts filter'), includes: [:operatingsystem,:subscriptions,:fact_values,:fact_names,:applicable_errata,:applicable_rpms]).each_record do |host| -%>
```

Comment:
sorry, I missed that in first review, could you please add spaces after commas here?